### PR TITLE
Add version var for SQL DLLs to make updates easier

### PR DESCRIPTION
**Pull request by photodude**
Pull Request for Issue Add version var for SQL DLLs to make future updates easier
backports similar changes from Joomla CMS

### Summary of Changes
- fix some extension ordering
- fix the php 5.6 sql extension naming

### Testing Instructions
code review / tests pass

### Documentation Changes Required
none

### CODE REVIEW

File: .appveyor.yml [hunk 1]
```diff
@@ -43,28 +43,37 @@ install:
...
+          Add-Content php.ini "`n"
+        }
+    - IF %PHP%==1 echo extension=php_pgsql.dll >> php.ini
     - IF %PHP%==1 echo extension=php_pdo_pgsql.dll >> php.ini
     - IF %PHP%==1 echo extension=php_pdo_sqlite.dll >> php.ini
     - IF %PHP%==1 echo extension=php_sqlite3.dll >> php.ini
+    - IF %PHP%==1 echo extension=php_pdo_mysql.dll >> php.ini
     - IF %PHP%==1 echo extension=php_mysqli.dll >> php.ini
-    - IF %PHP%==1 echo extension=php_pgsql.dll >> php.ini
+    - IF %PHP_VER_TARGET%==5.6 IF %PHP%==1 echo extension=php_mysql.dll >> php.ini
```
Comment (mbabker) [inline .appveyor.yml]: This one isn't required.  We don't have a driver here supporting `ext/mysql`.

Comment (photodude) [inline .appveyor.yml]: thanks, I overlooked that. fixed now.

**Comment by photodude** [comment]
Looks like we have a random failure in IteratorPgsqlTest::testForEach on php 5.3 travis. Failure is unrelated to this PR.
Might resolve itself with a rerun, still unrelated to this PR.

**Comment by wilsonge** [comment]
I reran it - it passed

**Comment by photodude** [comment]
thank you @wilsonge

**Comment by photodude** [comment]
I've added the fix to the chocolatey php installs so php 7.1 will stop failing. They are now defaulting to installing to versioned php directories so multiple php versions can be run side by side like ruby

**Comment by photodude** [comment]
@wilsonge @mbabker ping for merge consideration

**Pull request merged**